src/System.Management.Automation/engine/parser/Compiler.cs

```diff
@@ -4028,14 +4028,31 @@ public object VisitCommand(CommandAst commandAst)
                     var splatTest = element;
```

Comment:
@KirkMunro, your last commit had 1 failures in `PowerShell-CI-windows`
Enter-PSHostProcess tests.By Process Id.Can enter using NamedPipeConnectionInfo
```powershell
Exception calling "Invoke" with "0" argument(s): "The runspace state is not valid for this operation."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 136
136:                 $ps.AddScript('$pid').Invoke() | Should -Be $pwshId

```

Reply:
@PoshChan please restart Windows